ts/Extensions/Annotations/Popup.ts

```diff
@@ -865,143 +943,397 @@ H.Popup.prototype = {
                 .querySelectorAll('.' + PREFIX + 'tab-item-content');
 
             // ADD tab
-            this.addColsContainer(tabsContainers[0] as any);
+            this.addColsContainer(tabsContainers[0] as HTMLDOMElement);
+            indicators.addSearchBox.call(
+                this,
+                chart,
+                tabsContainers[0] as HTMLDOMElement
+            );
             indicators.addIndicatorList.call(
                 this,
                 chart,
-                tabsContainers[0] as any,
+                tabsContainers[0] as HTMLDOMElement,
```

Comment:
Why do you force cast. Does the tabsContainers do not contain HTMLElements?

Reply:
I get `Argument of type 'Element' is not assignable to parameter of type 'HTMLElement'.` and I thought that casting to `HTMLDOMElement` would be better than to `any`

Reply:
For sure it is, but it also disables most TypeScript validation (not as much as `any` though). I was wondering, if we could have `tabsContainers` limit itself to `HTMLDOMElement`. Or maybe `addIndicatorList` would be happy with `Element` type instead of `HTMLDOMElement` as well.